docs/user/setup.rst

```diff
@@ -24,15 +24,15 @@ systems Scala Native has been used with:
 
 Ubuntu::
 
-    $ sudo apt-get install clang libgc-dev
+    $ sudo apt-get install clang libgc-dev libunwind-dev
 
 macOS::
 
     $ brew install llvm bdw-gc
 
 FreeBSD::
 
-    $ pkg install llvm38 boehm-gc
+    $ pkg install llvm38 boehm-gc libunwind
 
 nix/nixOS::
```

Comment:
/cc @MasseGuillaume nix builds needs to be updated with new libunwind dependency.